### ISSUE TITLE
Please update this page - documentation for bulk user profile update and Taxonomy terms operations 

### ISSUE THREAD

**Issue by jagtapm1** [SharePoint/sp-dev-docs#9764]
As the ACS service is deprecated, please provide the details documentation how bulk user profile update can be done using Entra ID authentication and what scope required.
Similar for taxonomy automation - how to move terms/enable-disable terms operations can be done using app only permissions by registering the app in the Microsoft Entra ID.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 35a3017a-6d6c-237f-fc57-94247a8f7bf6
* Version Independent ID: 93413445-18fa-9245-ea64-1c1d789dce1e
* Content: [Granting access using SharePoint App-Only](https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azureacs)
* Content Source: [docs/solution-guidance/security-apponly-azureacs.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/solution-guidance/security-apponly-azureacs.md)
* Service: **sharepoint-online**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Ashlesha-MSFT** [comment]
Hello @jagtapm1,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@jagtapm1,
You're absolutely right that with the deprecation of ACS, there’s a growing need for updated documentation that covers these scenarios using the modern Azure AD app-only model. We'll raise this as a documentation gap internally
